src/Analysis/Ast/Impl/Documents/DocumentBuffer.cs

```diff
@@ -59,12 +110,21 @@ internal sealed class DocumentBuffer {
 
                     var start = NewLineLocation.LocationToIndex(lineLoc, change.ReplacedSpan.Start, _sb.Length);
                     var end = NewLineLocation.LocationToIndex(lineLoc, change.ReplacedSpan.End, _sb.Length);
+
+                    //Debug.Assert(end >= start);
```

Comment:
Do you want to remove these before merging? Seems like from debugging.

Reply:
Maybe... They are useful to debug changes that's why I left them behind. I can nix them.